docs/api/api/api.txt

```diff
@@ -32,34 +32,34 @@ GET /announcements
 
   Get a list of all announcements.
```

Comment:
You should remove the whole section

Reply:
Oh! I understand now. So no documentation at all about announcements. :+1: